script.motr/resources/lib/ws4py/LICENSE

```diff
@@ -0,0 +1,29 @@
+NOTICE: The version of ws4py included with kaithem has been modified.
```

Comment:
Sorry this is not allowed, please package ws4py dependency as a script.module and submit as part of another PR.

Reply:
OK, that was new to me :)
So I'll wrap this as an own module and create a second PR for that then?

Second how do I handle a module import in Kodi, is it the same?
```
from .ws4py import *
from .ws4py.client.threadedclient import WebSocketClient
```

Reply:
Exactly just the same, just make sure you add it as a dependency in addon.xml